pages/404.tsx

```diff
@@ -0,0 +1,27 @@
+import Header from '@/components/Header';
+import Section from '@/components/Section';
+import Link from 'next/link';
+import Image from 'next/image';
+import TakeMeHomeButton from '@/components/TakeMeHomeButton';
+
+export default function pageNotFound() {
```

Comment:
I think the naming for components would be in pascal case, so this should be `PageNotFound()`

Reply:
ahh ic ic